src/client/tile.h

```diff
@@ -70,6 +70,7 @@ class Tile : public LuaObject
     void drawCreatures(const Point& dest, LightView* lightView = nullptr);
     void drawTop(const Point& dest, LightView* lightView = nullptr);
     void drawTexts(Point dest);
+	void drawWidget(Point dest);
 
 public:
     void clean();
```

Comment:
destroy widget in clean

Reply:
Done, forgot to add it back when I was fighting with reference warnings months ago.